pkg/config/agentmanagement_test.go

```diff
@@ -138,3 +139,46 @@ func TestFullUrl(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "https://localhost:1234/example/api/namespace/test_namespace/remote_config?a=A&b=B", actual)
 }
+
+func TestBareConfig(t *testing.T) {
+	cfg := `
+protocol: http
+`
+	var am AgentManagement
+	err := yaml.UnmarshalStrict([]byte(cfg), &am)
+	assert.NoError(t, err)
+	assert.Equal(t, defaultConfig.CacheLocation, am.CacheLocation)
+}
+
+func TestUnmarshal(t *testing.T) {
+	cfg := `
+api_url: http://localhost:8080
+basic_auth:
+  username: test_user
+  password_file: /tmp/test/passfile
+protocol: http
+polling_interval: 1m
+remote_configuration:
```

Comment:
shall we include here the cache _location to make sure the default is properly overriden?